@here/harp-omv-datasource/lib/OmvDecodedTileEmitter.ts

```diff
@@ -531,6 +532,15 @@ export class OmvDecodedTileEmitter implements IOmvEmitter {
     ): void {
         this.processFeatureCommon(env);
 
+        const { projection, center, tileKey } = this.m_decodeInfo;
+
+        // get the bounds, and the size of the OMV Tile in its webmercator space.
+        const tileDataBounds = new THREE.Box3();
+        webMercatorTilingScheme.getWorldBox(tileKey, tileDataBounds);
```

Comment:
Cannot this be passed as part of ``OmvDecoder.DecodeInfo``? Otherwise, we're recalculating this for all features in a tile (when it's the same for all of them).

Reply:
We could but it doesn't really make any difference, also it is decoder-specific if the bounds are needed or not.